realsense2_camera/test/live_camera/rosci.py

```diff
@@ -0,0 +1,147 @@
+    print( '        -q, --quiet          Suppress output; rely on exit status (0=no failures)' )
+    print( '        -s, --stdout         Do not redirect stdout to logs' )
+    print( '        -r, --regex          Run all tests whose name matches the following regular expression' )
+    print( '        --list-tests         Print out all available tests. This option will not run any tests' )
+    print( '        --repeat <#>         Repeat each test <#> times' )
+    print( '        --config <>          Ignore test configurations; use the one provided' )
+    print( '        --device <>          Run only on the specified devices; ignore any test that does not match (implies --live)' )
+    print( '        --no-reset           Do not try to reset any devices, with or without a hub' )
+    print( '        --hub-reset          If a hub is available, reset the hub itself' )
+    print( '        --skip-disconnected  Skip live test if required device is disconnected (only applies w/o a hub)' )
+    print( 'Examples:' )
+    print( 'Running: python3.10 rosci.py -s' )
+    print( '    Runs all tests, but direct their output to the console rather than log files' )
+    print( 'Running: python3.10 rosci.py --list-tests' )
+    print( "    Will find all tests and print" )
+
+    sys.exit( 2 )
+
+def command(dev_name):
+       cmd =  ['pytest-3']
+       cmd += ['-s']
+       cmd += ['-m', ''.join(dev_name)]
+       cmd += ['-k', 'test_camera_imu_tests'] 
+       #cmd += ['-m', 'd415']
+       cmd += [''.join(dir_live_tests)]
+       cmd += ['--debug']
+       return cmd
+
+def run_test(cmd, dev_name, stdout=None, append =False):
+        handle = None
+        try:
+            stdout = stdout + os.sep + str(dev_name) + '_' + "test_camera_imu_tests" + ".log"
+            if stdout is None:
+                sys.stdout.flush()
+            elif stdout and stdout != subprocess.PIPE:
+                if append:
+                    handle = open( stdout, "a" )
+                    handle.write(
+                        "\n----------TEST-SEPARATOR----------\n\n" )
+                    handle.flush()
+                else:
+                    handle = open( stdout, "w" )
+
+            result = subprocess.run( cmd,
+                    stdout=handle,
+                    stderr=subprocess.STDOUT,
+                    universal_newlines=True,
+                    timeout=200,
+                    check=True )
+            if not result.returncode:
+                 log.i("---Test Passed---")
+        except Exception as e:
+             log.e("---Test Failed---")
+             log.w( "Error Exception:\n ",e )
+                         
+        finally:
+            if handle:
+                handle.close()      
+
+device_set = None
+try:
+    opts, args = getopt.getopt( sys.argv[1:], 'hvqr:st:',
+                                longopts=['help', 'verbose', 'debug', 'quiet', 'regex=', 'stdout', 'tag=', 'list-tags',
```

Comment:
Maybe only add option you support?
And on next PR's add options and implemantation?

Reply:
cleaned up in commit # 5. Not supporting any option for this PR. Test name and device name hard-coded for this PR.